tests/testImage.py

```diff
@@ -106,11 +106,17 @@ def testArrays(self):
             self.assertEqual(array1.shape[0], image2.getHeight())
             self.assertEqual(array1.shape[1], image2.getWidth())
             self.assertEqual(type(image3), cls)
+            array2 = image1.array
+            self.assertEqual(array1.shape, array2.shape)
```

Comment:
Why this test? If you want to assert these are identical I suggest using `numpy.testing.assert_array_equal` or `assertFloatsEqual`. I admit to a strong preference for the former because it is standard (thus any user can quickly look it up) and because it compares NaNs as equal.